get-changed-files/action.yml

```diff
@@ -0,0 +1,25 @@
+name: pr-filter
```

Comment:
Looks like you forgot to update action name and description; and output result description

Reply:
Thanks! fixed